stable/oauth2-proxy/templates/deployment.yaml

```diff
@@ -51,6 +55,12 @@ spec:
           - --authenticated-emails-file=/etc/oauth2-proxy/authenticated-emails-list
         {{- end }}
         {{- end }}
+        {{- with .Values.config.google }}
+        {{- if and .adminEmail (or .serviceAccountJson .existingSecret) }}
+          - --google-admin-email={{ .adminEmail }}
+          - --google-service-account-json=/google/service-account.json
```

Comment:
I can't see where is this `/google/` path mounted?

Reply:
A few line below under the `volumeMounts` property